docker-compose.yml

```diff
@@ -95,25 +95,42 @@ services:
       - ./graphql-api-gateway-poc/process.json:/usr/app/process.json
       - ./graphql-api-gateway-poc/log:/usr/app/log
     links:
-      - members-graphql-service
+      - inventory-management-graphql-service
+      - cart-graphql-service
```

Comment:
Not needed: "Links are not required to enable services to communicate - by default, any service can reach any other service at that service’s name. (See also, the Links topic in Networking in Compose.)" ([source](https://docs.docker.com/compose/compose-file/#links)). Though you could specify aliases for these hostnames, which would make it easier to access them in the container. 

If you're using this to specify dependencies, it's better to use [`depends_on`](https://docs.docker.com/compose/compose-file/#depends_on), which makes the purpose clearer.